test/.eslintrc.json

```diff
@@ -3,6 +3,7 @@
     "extends": [
         "../es6.js",
         "../addons/node.js",
-        "../addons/jest.js"
+        "../addons/jest.js",
+        "../addons/object-spread.js"
```

Comment:
I don't think we need this change either since it's already activated in the sample, see: https://github.com/moxystudio/eslint-config/blob/master/test/fixtures/samples/es7-object-spread/.eslintrc.json#L6